src/modules/layout/components/HeaderLanguageSelectorItem.js

```diff
@@ -10,8 +10,8 @@ import languageIcon from '../assets/language-icon.svg'
 import ReactTooltip from 'react-tooltip'
 import styled from 'styled-components'
 
-const InactiveImage = styled.img`
-  color: ${props => props.theme.colors.textSecondaryColor};
+const DisabledImage = styled.img`
+  color: ${props => props.theme.colors.textDisabledColor};
```

Comment:
This doesn't have any effect since its an image (Probably code written by me though :smile: )

Reply:
done

Reply:
You just removed the code right? Could you create a task to grey the icon out?